TCAT/Controllers/CustomNavigationController.swift

```diff
@@ -51,13 +63,34 @@ class CustomNavigationController: UINavigationController, UINavigationController
             Analytics.shared.log(payload)
         }
     }
+    
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        
+        // Add Notification Observers
+        NotificationCenter.default.addObserver(self,
+                                               selector: #selector(reachabilityChanged(notif:)),
+                                               name: .reachabilityChanged,
+                                               object: reachability)
+        guard let reach = reachability else {return}
+        do {
+            try reach.startNotifier()
+        } catch {
+            print("could not start reachability notifier")
```

Comment:
Are we going to use the "printClass"? If not, update the print message (correct punctuation).